doc/plugin_tutorial/README.md

```diff
@@ -1,5 +1,19 @@
 How to write plugins in Coq
 ===========================
```

Comment:
Github seems to be losing some of my comments :-(.

Ltac2 and elpi are not general purpose programming languages.  Sometimes you may need the richness of OCaml to do what you want.  New commands probably have to be in plugins.

We should mention that plugins are useful to add additional functions to Ltac2.

Reply:
> Github seems to be losing some of my comments :-(

Don't know if it's what you are suffering here, but what I sometime observe is comment linked to specific commits: I receive them by email but didn't found any way to retrieve them in the web interface. I don't know what creates them though.

Reply:
IIUC if you select a specific commit from the "Commits" tab and comment there, others will need to look at that specific commit to see the comment.  I think.